src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,24 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
```

Comment:
you can use destructuring if you want
```suggestion
export const TodoInfo: React.FC<Props> = ({ todo }) => {
  const { completed, title, user } = todo;
```